content/about/collaborators.md

```diff
@@ -0,0 +1,64 @@
++++
+# Homepage
+type = "blank"
+headless = true  # Homepage is headless, other widget pages are not.
+weight = 100
+title = "Collaborators and Funders"
+
+[design]
+  columns = "1"
++++
+
+<style>
+div.collaborator-box {
+  max-width: 300px;
+  min-height: 100px;
+  display: inline-block;
+  margin: 0px 2em;
+}
+</style>
+
+
+## Direct Funding
+
+<div class="collaborator-container">
+  <div class="collaborator-box">
+  <a class="collaborator-link" href="/posts/czi-core-support">
+    <img alt="CZI Logo" src="https://chanzuckerberg.com/wp-content/uploads/2018/09/Logo-Centered-1024-x-512-.png" />
+  </a>
+  </div>
+
+  <div class="collaborator-box">
+  <a class="collaborator-link" href="https://investinopen.org/blog/jrost-rapid-response-fund-awardees/">
+    <img style="width: 100px" src="https://investinopen.org/content/images/2020/09/ioi-logo-sq-1.png" />
+  </a>
+  </div>
+</div>
+
+## Collaborating Institutions
+
+<div class="collaborator-container">
+  <div class="collaborator-box">
+  <a class="collaborator-link" href="https://berkeley.edu/">
+    <img src="https://brand.berkeley.edu/wp-content/uploads/2016/11/primarylogo.png" alt="University of California, Berkeley" />
+  </a>
+  </div>
+
+  <div class="collaborator-box">
+  <a class="collaborator-link" href="https://www.icsi.berkeley.edu/icsi/">
+    <img src="https://www.icsi.berkeley.edu/icsi/sites/all/themes/icsi/logo.png" alt="International Computer Science Institute" />
+  </a>
+  </div>
+
+  <div class="collaborator-box">
+  <a class="collaborator-link" href="https://www.ldeo.columbia.edu/">
+    <img src="https://pangeo.io/_images/ldeo-logo.png" alt="Lamont Doherty Earth Observatory" />
+  </a>
+  </div>
+
+  <div class="collaborator-box">
+  <a class="collaborator-link" href="https://www.pims.math.ca/">
+    <img src="https://ssc.ca/sites/default/files/imce/webhorizfulllarge.png" alt="Pacific Institute for Mathematical Sciences" />
+  </a>
+  </div>
```

Comment:
It is probably appropriate to add UBC here now for both @colliand and I -- thoughts @colliand??

Reply:
Yes. I agree. Let's list UBC and PIMS.